spec/target_spec.rb

```diff
@@ -70,13 +71,15 @@ module Xcake
 
       settings = Xcodeproj::Project::ProjectHelper.common_build_settings(:debug, @target.platform, @target.deployment_target.to_s, @target.type, @target.language)
       settings.merge!({
-        "INFOPLIST_FILE" => "./$(PRODUCT_NAME)/Supporting Files/Info.plist"
+        "INFOPLIST_FILE" => "./$(PRODUCT_NAME)/Supporting Files/Info.plist",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.